test/extensions/filters/http/cache/cache_filter_test.cc

```diff
@@ -84,17 +88,30 @@ class CacheFilterTest : public ::testing::Test {
     ::testing::Mock::VerifyAndClearExpectations(&decoder_callbacks_);
   }
 
-  void testDecodeRequestHitWithBody(CacheFilterSharedPtr filter, std::string body) {
+  // This function assumes that there is a body created using std::string(body_size, 'a').
+  void testDecodeRequestHitWithBody(CacheFilterSharedPtr filter, uint64_t body_size) {
+    ASSERT(body_size > 0);
     // The filter should encode cached headers.
     EXPECT_CALL(decoder_callbacks_,
                 encodeHeaders_(testing::AllOf(IsSupersetOfHeaders(response_headers_),
                                               HeaderHasValueRef(Http::Headers::get().Age, age)),
                                false));
 
-    // The filter should encode cached data.
-    EXPECT_CALL(
-        decoder_callbacks_,
-        encodeData(testing::Property(&Buffer::Instance::toString, testing::Eq(body)), true));
+    // The filter should encode data in chunks sized according to the buffer limit.
+    int chunks_count = std::ceil(float(body_size) / float(buffer_limit_));
```

Comment:
const, here and other local variables that are not reassigned.

Reply:
Done. Thanks for the suggestion.